docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: '3.7'
+
+services:
+  web:
+    build: .
+    command: python manage.py runserver 0.0.0.0:8081
+    volumes:
+      - .:/app
+    ports:
+      - 8082:8081
+    depends_on:
+      - db
+  db:
+    image: library/postgres:12.0-alpine
+    volumes:
+    - postgres_data:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_USER=hello_django
+      - POSTGRES_PASSWORD=hello_django
+      - POSTGRES_DB=mis
```

Let's first define db and then web. Since web depends on db.
```suggestion
  db:
    image: library/postgres:12.0-alpine
    volumes:
      - postgres_data:/var/lib/postgresql/data/
    environment:
      - POSTGRES_USER=hello_django
      - POSTGRES_PASSWORD=hello_django
      - POSTGRES_DB=mis
      
  web:
    build: .
    command: python manage.py runserver 0.0.0.0:8081
    volumes:
      - .:/app
    ports:
      - 8082:8081
    depends_on:
       - db
```

---

Defined db first.